proto/ibc/lightclients/wasm/v1/genesis.proto

```diff
@@ -15,8 +15,6 @@ message GenesisState {
 // A contract's store key and code
```

Comment:
nit: outside the scope of this PR but proto docs should start with the message type name for both `GenesisContract` and `GenesisState`. 

Also, is the wrapper type `GenesisContract` needed? I think it would also be valid to use `repeated bytes` in `GenesisState`, but either works for me, it might be nice to keep the wrapper.

Reply:
Good point. Maybe just in case we can keep the wrapper for now, In case we add other fields in the future...